src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not a typical pattern for entity classes in Hibernate. Unless there's a specific requirement for cloning objects in your project, it's better to remove this to avoid confusion.